app/views/images/edit.html.erb

```diff
@@ -28,7 +28,7 @@ content_for :title, t(title_key, title: @edition.title_or_fallback)
   }
 ) do %>
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-column-two-thirds" id="alt-text">
```

Comment:
We should assign these IDs to the input component, not to presentation containers. By doing so we also get an automated focus on the specified field and, in our case, the associated guidance appearing on the right hand side.

Reply:
We didn’t do that as you can’t see the label, I hadn’t realised about the focus though. 

This suggestion has consequences for https://github.com/alphagov/content-publisher/pull/1078 too, so I’ll catch up with you about it later

Reply:
The component is build so that the label is displayed when the ID of the associated input is focus for this exact scenario.

Reply:
As discussed IRL - looks like there is a bug that stops the scroll behaviour that you're hoping for, so we concluded this is the best user experience for now.

Alex may work on resolving this later.